src/__test__/components/data-exploration/generic-gene-table/SelectionActions.test.jsx

```diff
@@ -127,8 +126,8 @@ describe('SelectionIndicator', () => {
     const button = component.find(Button);
     const text = component.find(Text);
 
-    // There should be one button.
-    expect(button.length).toEqual(1);
+    // There should be two buttons.
+    expect(button.length).toEqual(2);
 
     // An 'export as csv...' button
     expect(button.at(0).childAt(0).text()).toEqual('Export as CSV ...');
```

Comment:
Can you add a check for the correct text rendering of the 2nd button? Following the test here.